spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -52,22 +52,22 @@ On the other hand, if an entity class is annotated with an entity-defining annot
 This allows an entity class to be compiled with multiple entity-defining annotations, each targeting a different Jakarta Data provider, and have the provider automatically disambiguated at runtime based on the availability of the entity-defining annotations.
 ====
 
-=== Provider Name
+=== Provider name
 
 The entity-defining annotation type is the preferred way to avoid conflicts between Jakarta Data providers.
 But when multiple Jakarta Data providers support the same entity-defining annotation, the application must disambiguate the Jakarta Data provider for a repository by explicitly specifying the `provider` attribute of the `Repository` annotation.
 
 A Jakarta Data provider must ignore every repository which specifies the name of a different `provider` in its `@Repository` annotation.
 
-== Interoperability with other Jakarta EE Specifications
+== Interoperability with other Jakarta EE specifications
 
 This section discusses interoperability with related Jakarta EE footnote:[Jakarta EE Platform 11, https://jakarta.ee/specifications/platform/11/] specifications. When operating within a Jakarta EE product, the availability of other Jakarta EE technologies depends on whether the Jakarta EE Core profile, Jakarta EE Web profile, or Jakarta EE Platform is used.
 
 === Jakarta Contexts and Dependency Injection
 
 Contexts and Dependency Injection footnote:[Jakarta Contexts and Dependency Injection 4.1, https://jakarta.ee/specifications/cdi/4.1/] (CDI) is a specification in the Jakarta EE Core profile that provides a powerful and flexible dependency injection framework for Java applications. CDI provides a programming model based around decoupled components with container-managed lifecycles and container-injected dependencies, enabling loose coupling and promoting modular and reusable code.
 
-==== CDI Dependency Injection
+==== CDI dependency injection
```

Comment:
contexts and dpendency injection

Reply:
@otaviojava WDYM? That section is talking about injection.

Reply:
never mind!